apps/onlyoffice.sh

```diff
@@ -48,6 +48,18 @@ sudo bash onlyoffice.sh"
     exit 1
 fi
 
+# check if apache2 evasive-mod is enabled and disable it because of compatibility issues
+if [ "$(Apache2ctl -M | grep evasive)" != "" ]
+then
+    msg_box "Please note: if you continue, the ddos protection for webservices will get disabled now, because of compatibility issues with onlyoffice."
+    if [[ "no" == $(ask_yes_or_no "So do you want to continue?")  ]]
+        then
+            exit 1
```

Comment:
Just add `sleep 0.1` here, if you exit the whole script exits.

Reply:
you have already done this ..👍